.github/policies/issue.add-no-recent-activity.yml

```diff
@@ -21,10 +21,9 @@ configuration:
       - addLabel:
           label: no-recent-activity
       - addReply:
-          reply: >-
+          reply: |-
```

Comment:
I believe this is the preferred form for handling blank lines in a multiline string in YAML.